bigtable/quickstart/README.md

```diff
@@ -0,0 +1,78 @@
+# Cloud Bigtable Quickstart in Go
+
+This is a simple application that demonstrates using the [Google Cloud APIs Go
+Client Library](https://github.com/GoogleCloudPlatform/google-cloud-go) to connect
+to a Cloud Bigtable instance and read a row from a table.
+
+## Before you begin
+
+1. [Select or create][projects] a Cloud Platform project.
+
+1. Enable [billing][billing] for your project.
+
+1. Enable the Cloud Bigtable API][enable_api].
+
+    Note: The quickstart performs an operation on an existing table.
+    If you require your code to create instances or tables,
+    the [Admin API](https://console.cloud.google.com/flows/enableapi?apiid=bigtableadmin.googleapis.com)
+    must be enabled as well.
+
+1. [Set up authentication with a service account][auth] so you can access the API from your local workstation.
+
+1. Follow the instructions in the [user documentation](https://cloud.google.com/bigtable/docs/creating-instance) to
+create a Cloud Bigtable instance (if necessary).
+
+1. Follow the [cbt tutorial](https://cloud.google.com/bigtable/docs/quickstart-cbt) to install the
+cbt command line tool.
+Here are the cbt commands to create a table, column family and add some data:
+```
+   cbt createtable my-table
+   cbt createfamily my-table cf1
+   cbt set my-table r1 cf1:c1=test-value
+```
+
+[projects]: https://console.cloud.google.com/project
+[billing]: https://support.google.com/cloud/answer/6293499#enable-billing
+[enable_api]: https://console.cloud.google.com/flows/enableapi?apiid=bigtable.googleapis.com
+[auth]: https://cloud.google.com/docs/authentication/getting-started
+
+
+## Running the quickstart
+
+The [Quick start](main.go) sample shows a basic usage of the Bigtable client library: reading rows from a table.
```

Comment:
The [quickstart](main.go) sample shows how to read rows from a table with the Bigtable client library.